docs/snippets_test.py

```diff
@@ -221,7 +264,9 @@ def test_canonicalize_printed_line():
                           '[0.]']}) == 1
 
 
-def assert_code_snippet_executes_correctly(snippet: str, state: Dict):
+def assert_code_snippet_executes_correctly(snippet: str,
+                                           state: Dict,
+                                           line_number: int = None):
```

Comment:
Is this ever actually invoked without a `line_number`?

Reply:
Yes, in `test_assert_code_snippet_executes_correctly` (and only in these tests). Should I change the tests to all have fake line numbers?

Reply:
Eh, either way is fine.